tests/boot/grub_test_snapshot.pm

```diff
@@ -1,15 +1,15 @@
 # SUSE's openQA tests
 #
 # Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2016 SUSE LLC
+# Copyright © 2012-2017 SUSE LLC
```

Comment:
2018 already :)

Reply:
you know you are reviewing a PR from 2017-10?